internal/mage/util/engine.go

```diff
@@ -213,24 +215,33 @@ func devEngineContainer(c *dagger.Client, arch string, version string, opts ...D
 
 	container := c.Container(dagger.ContainerOpts{Platform: dagger.Platform("linux/" + arch)}).
 		From("alpine:"+alpineVersion).
+		// NOTE: wrapping the apk installs with this time based env ensures that the cache is invalidated
+		// once-per day. This is a very unfortunate workaround for the poor caching "apk add" as an exec
+		// gives us.
+		// Fortunately, better approaches are on the horizon w/ Zenith, for which there are already apk
+		// modules that fix this problem and always result in the latest apk packages for the given alpine
+		// version being used (with optimal caching).
+		WithEnvVariable("DAGGER_APK_CACHE_BUSTER", fmt.Sprintf("%d", time.Now().Unix()/86400)).
```

Comment:
nit: using `(time.Time).Truncate` makes this a bit easier to grok

```suggestion
		WithEnvVariable("DAGGER_APK_CACHE_BUSTER", fmt.Sprintf("%d", time.Now().Truncate(24 * time.Hour).Unix())).
```

or:

```suggestion
		WithEnvVariable("DAGGER_APK_CACHE_BUSTER", time.Now().Truncate(24 * time.Hour).String()).
```

Reply:
Nice, I missed that `Truncate` existed! Updated